test/studies/IMSuite/vertexColor/vertexColoring-serial.chpl

```diff
@@ -0,0 +1,319 @@
+ /* This file is part of IMSuite Benchamark Suite.
+
+    This file is licensed to You under the Eclipse Public License (EPL);
+    You may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+    http://www.opensource.org/licenses/eclipse-1.0.php
+
+   (C) Copyright IMSuite 2013-present.
+ */
```

Comment:
I think the answer to this question depends on what the EPL says about derivative works.  Specifically, if this version was created by translating / transliterating a version distributed under the EPL and the EPL specifies that such copyright notices must be preserved in derivative works then I think we would need to keep such a notice in the file.

If that's the case, then I think we should put a prefix comment in front of this that says something like:  "This port of this benchmark was written in Chapel by transliterating [add some very clear reference to original code including maybe a URL?] which was distributed under the following license:"  And then I'd preserve the header that's noted above, though I'd probably remove the "This file is part of ..." line since it isn't accurate anymore (and presumably not required by the EPL).